results/user-testing-results.md

```diff
@@ -0,0 +1,138 @@
+
+#### Flexible command palette search/filter behavior
+
+Because several participants relied heavily on the command palette to complete tasks, the limit of its filtering arose several times. From the user end, the command palette feels like it searches only for direct matches within the command list for whatever the user types in. If a user knows exactly what they are looking for, they often have success. Participants were sometimes able to quickly and flawlessly locate a desired command. But when they were exploring JupyterLab’s command options, unable to remember the exact name of a command, made a typo, or were unfamiliar with a task many participants had to give up using the command palette in order to complete the task. Depending on the assistive tech they use, typos can also be nontrivial to detect for blind or visually impaired users. Having a fuzzy search or other typo/error support rather than only the direct filter would help the command palette continue to be the boon many participants wanted it to be.
+
+## For future tests
+
+Reflecting on the findings and gaps in this set of tests, the following are questions to revisit in the future.
+
+- How might a screen reader user complete common notebook tasks in JupyterLab?
+- What are JupyterLab’s strengths and weaknesses when it comes to authoring notebooks and other documents with assistive tech?
+- What more can we learn about navigating JupyterLab, especially with a variety of assistive tech? What are good examples of navigation that support disabled users?
+- What are the full range of notebook cell output types and what feedback to disabled users have on each type?
+- What is the real time collaboration experience in JupyterLab like for assistive tech users? This question requires JupyterLab collaboration modes to be more stable before it can be tested.
+
+As mentioned in the Limits section, the sample we were able to test with for these sessions was not as comprehensive of varied disabilities and assistive tech as we would like. Recruitment for these sessions was done within existing JupyterLab community spaces and was mostly self-selected; future tests would likely benefit from a more selective sample and outreach to disability communities where appropriate. This should be done with careful consideration as the history of people who make products interrupting disability communities and asking for labor is a fraught one.
+
+A usability testing style script was chosen for this testing session because of the the need for scoped and actionable accessibility feedback in JupyterLab, project timeline, available team, and other logistical constraints. After these results, we can recommend that observational/[field studies](https://www.nngroup.com/articles/field-studies/) would be useful in understanding the broader, holistic accessibility issues within JupyterLab’s user experience that a study like this is not suited to survey.
```

Comment:
```suggestion
A usability testing style script was chosen for this testing session because of the need for scoped and actionable accessibility feedback in JupyterLab, project timeline, available team, and other logistical constraints. After these results, we can recommend that observational/[field studies](https://www.nngroup.com/articles/field-studies/) would be useful in understanding the broader, holistic accessibility issues within JupyterLab’s user experience that a study like this is not suited to survey.
```